static/resources/harness.js

```diff
@@ -1337,7 +1337,7 @@
       }
 
       if ("serviceWorker" in navigator) {
-        window.__workerCleanup();
+        await window.__workerCleanup();
```

Comment:
Can't use `async`/`await` in this file; it must stay ES3 compatible.

```suggestion
        window.__workerCleanup();
```